pkg/kubefed2/enable/enable.go

```diff
@@ -234,11 +235,24 @@ func CreateResources(cmdOut io.Writer, config *rest.Config, resources *typeResou
 	if err != nil {
 		return errors.Wrap(err, "Failed to create crd clientset")
 	}
-	_, err = crdClient.CustomResourceDefinitions().Create(resources.CRD)
-	if err != nil {
-		return errors.Wrapf(err, "Error creating CRD %q", resources.CRD.Name)
+
+	existingCRD, err := crdClient.CustomResourceDefinitions().Get(resources.CRD.Name, metav1.GetOptions{})
+	if apierrors.IsNotFound(err) {
+		_, err = crdClient.CustomResourceDefinitions().Create(resources.CRD)
+		if err != nil {
+			return errors.Wrapf(err, "Error creating CRD %q", resources.CRD.Name)
+		}
+		write(fmt.Sprintf("customresourcedefinition.apiextensions.k8s.io/%s created\n", resources.CRD.Name))
+	} else if err != nil {
+		return errors.Wrapf(err, "Error getting CRD %q", resources.CRD.Name)
+	} else {
```

Comment:
Please avoid using `else` here and the unnecessary indentation it requires.

Reply:
I think it needed. Or else, we will I think it needed. Or else, after creating CRD, we will update again. It is not good to me from logic.

Reply:
Ah, my mistake.